src/sentry/static/sentry/app/views/usageStats/index.tsx

```diff
@@ -84,7 +132,14 @@ class OrganizationStats extends React.Component<Props> {
       willUpdateRouter: true,
     }
   ): LocationDescriptorObject => {
+    Object.keys(nextState).forEach(k => {
+      if (!PAGE_QUERY_PARAMS.includes(k)) {
+        throw new Error('UsageStats: Unrecognized key for page query params');
```

Comment:
This seems like it could end up causing the page to fail if URLs contain additional parameters. It would be safer to only pick out what you need from the URL parameters instead.